apiserversdk/proxy.go

```diff
@@ -25,13 +30,16 @@ func NewMux(config MuxConfig) (*http.ServeMux, error) {
 		return nil, err
 	}
 	proxy := httputil.NewSingleHostReverseProxy(u)
-	if proxy.Transport, err = rest.TransportFor(config.KubernetesConfig); err != nil { // rest.TransportFor provides the auth to the K8s API server.
+	baseTransport, err := rest.TransportFor(config.KubernetesConfig)
+	if err != nil { // rest.TransportFor provides the auth to the K8s API server.
 		return nil, err
 	}
+	proxy.Transport = newRetryRoundTripper(baseTransport, 3)
```

Comment:
make retry count a constant, and make sure it's compatible with v1.
https://github.com/machichima/kuberay/blob/a321fbc81220c75c0d52b4ebab337810f7cd4f50/apiserver/pkg/util/config.go#L28-L37
we should at least followup a PR to unify these retry configs between v1 and v2